### DISCUSSION TITLE
Javascript instead of Typescript

### DISCUSSION THREAD

**Discussion by krd** [infinitered/ignite#1889] [Q&A] [answered]
Hello.  I love the ignite-cli and all it has to offer.  I'm wondering if it is possible to bootstrap a 'new' project to use Javascript instead of Typescript.  Fingers crossed that I can... 

Thanks

**Comment by jamonholmgren** [comment, marked as answer]
Hi @krd ! While it's possible to remove TypeScript from an Ignited project, there are quite a few steps, so you're probably better off initializing an empty React Native project and bringing over Ignite concepts as needed. Sorry I don't have a better answer for this! You can also just rename all of your `.ts|.tsx` files to `.js` (NOT `.jsx`) and make sure [allowJs](https://github.com/infinitered/ignite/blob/a811ced9ad4c8b1e95cd3cf93bda49822f33b7d8/boilerplate/tsconfig.json#L3) is `false`, remove all type annotations and definitions, and it should work as expected.

**Reply by krd** [reply]
Thank you for the quick reply.  I really love the speed that the cli gives to you.  Reminds me a lot of Rails.  I'll try making the changes you suggested.  thanks again

**Reply by shaharhalutz** [reply]
Hi @krd , just curious did you try the second option ? (renaming all files etc .. )
did it work for you ?
